src/test/emulators/functionsEmulator.spec.ts

```diff
@@ -0,0 +1,121 @@
+import { expect } from "chai";
+import { FunctionsEmulator, FunctionsRuntimeInstance } from "../../emulator/functionsEmulator";
+import * as supertest from "supertest";
+import { FunctionRuntimeBundles, TIMEOUT_LONG, TIMEOUT_MED } from "./fixtures";
+import * as logger from "../../logger";
+import { FunctionsRuntimeBundle } from "../../emulator/functionsEmulatorShared";
+import * as express from "express";
+
+// Uncomment this to enable --debug logging!
```

Comment:
Could we gate this on an environment variable instead?

Reply:
Sure I suppose